### PR TITLE
page transitions

### LINKED ISSUE #82: page transitions
**Is your feature request related to a problem? Please describe.**
Will provide smoother transitions between grid states

**Describe the solution you'd like**
Styled components

### PR DESCRIPTION
**Pull request by gonzalespaulb**
## Changes
1. Added page transitions between app states

## Purpose
Seamless UX navigation

## Approach
Styled components and using the mounting and dismounting properties of useState

Closes #82

### CODE REVIEW

**Review by connielion — APPROVED** [review]
Awesome~

**Review by BlackCubes — APPROVED** [review]
LGTM!

File: src/components/RecipeDetails.tsx [hunk 1]
```diff
@@ -56,6 +65,11 @@ const RecipeDetailsCard = styled.div`
   position: relative;
   padding-left: 70px;
   overflow: hidden;
+  opacity: 0;
+  animation-name: ${recipeDetailsTransition};
+  animation-duration: 2.5s;
+  animation-fill-mode: forwards;
+  animation-iteration-count: 1;
```
Comment (krferrell) [inline src/components/RecipeDetails.tsx:72]: I believe you can shorthand this to something like "animation: ${textFlash} .8s linear infinite alternate;"

**Review by krferrell — APPROVED** [review]
Looking good!

**Pull request merged**